integrationTests/knexfile.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  development: {
+    client: 'postgresql',
+    connection: {
+      host: 'localhost',
+      database: 'hitech_apd',
```

Comment:
could we change the database to be test specific? like hitech_apd_test or hitech_apd_cypress?

Reply:
not without restarting the containers.  The API is already up and connected to a DB before we get going.

Reply:
that is a good idea though but it does seem like it would take a bit of work and/or duplication to create another seeded db + have the api connect to it. maybe a future enhancement?